app/src/components/input/input.types.ts

```diff
@@ -0,0 +1,16 @@
+import { ChangeEvent } from "react";
+
+export interface InputAttributes {
```

Comment:
Should we follow the custom of pre-fixing interfaces with "I"? I think it would help with clarity?

Reply:
Yes that sounds like a good idea